MANIFEST.in

```diff
@@ -1 +1,2 @@
 include LICENSE
+graft hangups/test
```

Comment:
Looks like this inadvertently includes `__pycache__`. Should it be `recursive-include` with a pattern instead?

Reply:
Afaik this is only used for publishing sources to PyPI:
https://docs.python.org/3.7/distutils/commandref.html#creating-a-source-distribution-the-sdist-command

Reply:
Yes, but I don't think we want the source distribution to include `pyc` files.

Reply:
Does this seem reasonable:
https://github.com/chardet/chardet/commit/d7c7343ca89df6ec9c329b6cdccfdcab8b027c12

Reply:
Yes, looks good.